tests/test_portal.py

```diff
@@ -264,27 +268,30 @@ def test_redirect_validation(self):
 
         client = self.add_client()
         client_url = client._redirect_uris
-        local_url = "http://{}/home?test".format(self.app.config.get('SERVER_NAME'))
+        local_url = "http://{}/home?test".format(
+            self.app.config.get('SERVER_NAME'))
         invalid_url = 'http://invalid.org'
 
         # validate redirect of /website-consent-script GET
         response = self.client.get('/website-consent-script/{}?redirect_url='
-                             '{}'.format(TEST_USER_ID, local_url))
+                                   '{}'.format(TEST_USER_ID, local_url))
         assert response.status_code == 200
 
         response2 = self.client.get('/website-consent-script/{}?redirect_url='
-                              '{}'.format(TEST_USER_ID, invalid_url))
+                                    '{}'.format(TEST_USER_ID, invalid_url))
         assert response2.status_code == 401
 
         # validate redirect of /login/<provider> GET
-        response3 = self.client.get('/login/TESTING?user_id={}&next='
-                              '{}'.format(TEST_USER_ID, client_url),
-                              follow_redirects=True)
+        response3 = self.client.get(
+            '/login/TESTING?user_id={}&next={}'.format(
```

Comment:
Can you try passing the querystring arguments as a dict with `query_string`?
eg:
```
self.client.get(
    '/login/TESTING',
    query_string={'user_id': TEST_USER_ID, 'next': client_url},
    follow_redirects=True,
)
```